ddl/db_partition_test.go

```diff
@@ -1059,7 +1055,26 @@ func (s *testIntegrationSuite7) TestExchangePartitionExpressIndex(c *C) {
 	tk.MustExec("alter table nt1 drop index idx;")
 	tk.MustExec("alter table nt1 add index idx((concat(a, b)));")
 	tk.MustGetErrCode("alter table pt1 exchange partition p0 with table nt1;", tmysql.ErrTablesDifferentMetadata)
+}
+
+func (s *testIntegrationSuite7) TestExchangePartitionAutoRandom(c *C) {
```

Comment:
We need to move this test to `ddl/serial_test.go` to keep serial execution because the global config is not goroutine-safe:
```
func (s *testSerialSuite) TestAutoRandom(c *C)
```

(After the `auto_random` feature is moved out of `experimental`, this test can be placed here.)